app/middlewares/validatonSchemas/users.js

```diff
@@ -0,0 +1,54 @@
+const { user: UserModel } = require('../../models/index');
+
+const UserSchemas = module.exports;
+
+const providedAndNotEmpty = 'should be provided and not empty';
+
+UserSchemas.createUserSchema = {
+  first_name: {
+    in: 'body',
+    optional: false,
+    notEmpty: true,
+    errorMessage: providedAndNotEmpty
+  },
+  last_name: {
+    in: 'body',
+    optional: false,
+    notEmpty: true,
+    errorMessage: providedAndNotEmpty
+  },
+  email: {
+    in: 'body',
+    optional: false,
+    notEmpty: true,
+    errorMessage: providedAndNotEmpty,
+    isEmail: {
+      errorMessage: 'should be a valid email',
```

Comment:
Same comment, put this messages into a constants file or maybe into the errors.js file. I prefer a constants file. What do you think?

Reply:
It's also better for me in a constants file